src/constants.js

```diff
@@ -0,0 +1,60 @@
+export const TASKS_MANAGEMENT_MAIN_MENU_CONTRIBUTION_KEY = 'tasksManagement.MainMenu';
+
+export const DEFAULT_DEBOUNCE_TIME = 500;
+export const DEFAULT_PAGE_SIZE = 10;
+export const EMPTY_STRING = '';
+export const HYPHEN = '-';
+export const ROWS_PER_PAGE_OPTIONS = [10, 20, 50, 100];
+export const CONTAINS_LOOKUP = 'Icontains';
+
+export const TASK_STATUS = {
+  RECEIVED: 'RECEIVED',
+  ACCEPTED: 'ACCEPTED',
+  COMPLETED: 'COMPLETED',
+  FAILED: 'FAILED',
+};
+
+export const POLICY_STATUS = {
+  ALL: 'ALL',
+  ANY: 'ANY',
+  N: 'N',
+};
+
+export const TASK_COLUMNS = {
+  SOURCE: 'source',
+  TYPE: 'type',
+  ENTITY: 'entity',
+  ASSIGNEE: 'assignee',
+  BUSINESS_STATUS: 'businessStatus',
+  STATUS: 'status',
+};
```

Comment:
Recently while we were working with Kamil on the updated solution for the Tasks management some of the field names has changed to reflect their purposes better. Can you check with him if any frontend elements require adjustments?

Reply:
Moving all benefit plan logic to the other modules will be handled in scope of the [CM-139](https://openimis.atlassian.net/browse/CM-139). I just created a dummy data based on the provided mockups to start working on tasks feature.